app/js/questions_controller.js

```diff
@@ -43,6 +45,29 @@
     var tab_panels = container.querySelectorAll('[role="tabpanel"]');
     toggle_state(tab_panels, 'aria-hidden');
 
+    load_active_questions();
+  }
+
+  function load_active_questions() {
```

Comment:
We should move this function lower in the file to not break the visual space between `set_active_tab` and `add_tab_widget_handler`